src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -79,26 +74,43 @@ public class Drive extends SubsystemBase implements Vision.VisionConsumer {
               Math.hypot(TunerConstants.BackLeft.LocationX, TunerConstants.BackLeft.LocationY),
               Math.hypot(TunerConstants.BackRight.LocationX, TunerConstants.BackRight.LocationY)));
 
-  // PathPlanner config constants
-  private static RobotConfig PP_CONFIG;
+  // Main Robot
+  // Weight: 48.35295 KG
+  // MOI: 15.098568
+  // Wheel Radius: TunerConstants
+  // Max Speed: 4.4 mps
+  // Wheel COF: 1.2
+  // Motor Type: KrakenX60 Foc
+  // Current Limit: TunerConstants
+
+  // Second Robot
+  // Weight: ~ 30.54945 KG
+  // MOI: ~ 9.539292
+  // Wheel Radius: Tuner Constants
+  // Max Speed: ~ 4.4 mps
+  // Wheel COF: 1.2
+  // Motor Type: Falcon500 Foc
+  // Current Limit: TunerConstants
+
+  public static RobotConfig PP_CONFIG;
 
   public static final DriveTrainSimulationConfig mapleSimConfig =
       DriveTrainSimulationConfig.Default()
-          .withRobotMass(DrivetrainConstants.ROBOT_MASS)
+          .withRobotMass(Kilograms.of(30))
           .withCustomModuleTranslations(getModuleTranslations())
           .withGyro(COTS.ofPigeon2())
-          .withBumperSize(Meters.of(0.876), Meters.of(0.876))
+          .withBumperSize(kBumperSize, kBumperSize)
           .withSwerveModule(
               new SwerveModuleSimulationConfig(
-                  DCMotor.getKrakenX60(1),
-                  DCMotor.getKrakenX60(1),
+                  DCMotor.getKrakenX60Foc(1),
+                  DCMotor.getKrakenX60Foc(1),
                   TunerConstants.FrontLeft.DriveMotorGearRatio,
                   TunerConstants.FrontLeft.SteerMotorGearRatio,
                   Volts.of(TunerConstants.FrontLeft.DriveFrictionVoltage),
                   Volts.of(TunerConstants.FrontLeft.SteerFrictionVoltage),
                   Meters.of(TunerConstants.FrontLeft.WheelRadius),
                   KilogramSquareMeters.of(TunerConstants.FrontLeft.SteerInertia),
-                  DrivetrainConstants.WHEEL_COF));
+                  1.2));
```

Comment:
Need to resolve constants either to select between either robot config or just use the main robot's

Reply:
We can't use a robot config, that breaks everything for some reason, so we have to use the .getGUIConfig. so there really isn't a great solution here